ruby/ql/src/queries/security/cwe-020/MissingFullAnchor.qhelp

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE qhelp PUBLIC
+"-//Semmle//qhelp//EN"
+"qhelp.dtd">
+<qhelp>
+
+	<overview>
+		<p>
+            Regular expressions in Ruby can use anchors to match the beginning and end of a string. 
+            However, if the <code>^</code> and <code>$</code> anchors are used, 
+            the regular expression can match a single line of a multi-line string.
+		</p>
+	</overview>
+
+	<recommendation>
+		<p>
+            Use the <code>\A</code> and <code>\z</code> anchors to match the beginning and end of a string, 
+            as these will always match the beginning and end of the string, even if the string contains newlines.
```

Comment:
```suggestion
            Use the <code>\A</code> and <code>\z</code> anchors since these anchors will always match the beginning and end of the string, even if the string contains newlines.
```

Small suggestion to make this sentence less wordy. Do you think this is still clear enough?